lib/provisioning/Provisioner.js

```diff
@@ -12,9 +12,26 @@ function Provisioner(ircBridge) {
     let as = this._ircBridge.getAppServiceBridge().appService;
     let self = this;
 
+    if (!(as.app.use && as.app.get && as.app.post)) {
+        log.warn('Could not start provisioning API');
+        return;
```

Comment:
It's not in the config, but it probably should be. This also begs the question, should the POST and GET endpoints not return something useful when provisioning hasn't been turned on? Something like, "Provisioning hasn't been turned on..."